base/file.jl

```diff
@@ -496,13 +509,6 @@ end
 end # os-test
 
 
-"""
-    tempdir()
-
-Obtain the path of a temporary directory (possibly shared with other processes).
-"""
-tempdir()
-
 """
```

Comment:
In the `tempname` doc string below, it might be clearer to add, "This can lead to security holes." and recommend opening the file with `O_EXCL`, if this is a concern (in addition to the suggestion to use `mktemp`).

Aside: the windows implementation of `mktemp` appears to be wrong because it fails to heed this warning.